### ISSUE TITLE
[$30][MSFT-77] [Register] While tabbing the 'Login' hyperlink selection is not visible

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2791]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member/registration
2. Focus the cursor on the first name field and click tab till the selection reaches to Login

**Expected Result**
While tabbing the current selection should be marked by a rectangle. Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible.

**Actual Result**
User should be able to select the login options via the keyboard tab key

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: AA
**WCAG Success Criterion**: 2.4.7 Focus Visible
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F40

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096277 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096277 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096277 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu PR https://github.com/appirio-tech/accounts-app/pull/224

**Comment by PrakashDurlabhji** [comment]
@veshu @tosha5252 @mishacucicea PR updated

**Comment by mishacucicea** [comment]
@PrakashDurlabhji Please take a look at the login page and change styling so that focusing will also change put the links in a selection rectangle.
![image](https://user-images.githubusercontent.com/1721199/61835141-47333d80-ae83-11e9-9a14-f1edd685d6aa.png)

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097399 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097399 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097399 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
Already fixed by PR https://github.com/appirio-tech/accounts-app/pull/266

@veshu ticket prize yet to be doubled. Thanks!

**Comment by Oanh-and-only-Oanh** [comment]
@veshu @r0hit-gupta, doubling prize amount.

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097399 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by mishacucicea** [comment]
@r0hit-gupta - I'm sorry, but the PR you're linking is fixing the Login page, and this ticket is about the Register page, which is not working in the referenced PR (I tested it).

Try to stick to the rule "1 ticket = 1 PR", which makes it easy for everyone.

**Comment by nithyaasworld** [comment]
@mishacucicea Can you provide the PR # that fixes this issue?

**Comment by tosha5252** [comment]
Verified working

OS: Windows 10
Browser: Chrome and Edge

**Comment by nkumar-topcoder** [comment]
@r0hit-gupta based on PR 266, this issue #2791 and #2800 appears to have common fix. Can you please confirm. FYI @mishacucicea @nithyaasworld

**Comment by r0hit-gupta** [comment]
@mishacucicea @nkumar-topcoder yes, both the issues have a common fix. This issue can be said to be a subset of issue https://github.com/topcoder-platform/community-app/issues/2800 .

The CSS used in PR https://github.com/appirio-tech/accounts-app/pull/266 applies to the `LOG IN` & `JOIN NOW` hyperlink for both the login and registration pages. 

I am able to see the changes correctly on the test URL - https://accounts.topcoder-dev.com/member/registration

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097399 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@nithyaasworld @mishacucicea  my PR also solved it

**Comment by ajefts** [comment]
Verified this is working in prod.

**Comment by crazyk07** [comment]
Solved

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097399<br/><br/>```This is an automated message for crazyk via Topcoder X```
